doc/sai-ptf/pcbb_test_plan.md

```diff
@@ -0,0 +1,346 @@
+- decap_dscp_remap: This verifies on decapsulation, the DSCP inner field is preserved end-to-end and mapping to the expect dscp value base on the DSCP map.
+- decap_dscp_priority:  This verifies on decapsulation, the DSCP inner field is preserved end-to-end and mapping to the expect priority value base on the DSCP map.
+- decap_dscp_queue:  This verifies on decapsulation, the DSCP inner field is preserved end-to-end and mapping to the expect queue value base on the DSCP map.
+- decap_pfc_pause: This verifies the pfc frame generated as expected in decap.
+- decap_ecn_no_congestion: This verifies the ecn generated as expected in decap when no congestion happens.
+- decap_ecn_congestion: This verifies the ecn generated as expected in decap when congestion happens.
+- decap_global_dscp_to_tc_map: This verifies on decapsulation, the DSCP inner field is preserved end-to-end and mapping to the expect dscp value base on the DSCP map.
+
+
+    We will send a decapsulated packet from LAG1 and expect a decapsulated packet on port1
+
+  Egress side[port1]           |          Ingress side[lag1]
+  -----------------------------|-------------------------------------
+  ipv4's falls in 192.168.1.0     |        ipv4's falls in 10.1.0.0
+  ipv6's falls in fc02::1:0  |   ipv6's falls in fc00:1::
+### Testing Data Packet <!-- omit in toc --> 
+
+This test should cover the data in the table below
+
+DSCP_MAP_TABLE
+|DSCP(Base on Inner)|TC to verify| Outgoing Queue(Port)|Priority(Tunnel)|
+| ---- | ---- | -|-|
+|0-2|1|1|0|
+|3|3|3|2|
+|4|4|4|6|
+|5-7|1|1|0|
+|8|0|0|0|
+|9-32|1|1|0|
+|33|8|1|0|
+|34-45|1|1|0|
+|46|5|5|0|
+|48|7|7|7|
+|49-63|1|1|0|
+
+For ECN testing
+ECN_NON_CGN_TABLE:
+|Outer_DSCP|Outer_ECN|Inner_DSCP|Inner_ECN|Decap_DSCP|Decap_ECN|
+|-|-|-|-|-|-|
+|2|3|3|3|3|3|
+|2|2|3|2|3|2|
+|2|1|3|1|3|1|
+|2|3|3|2|3|3|
+|2|2|3|3|3|2|
+|6|3|4|3|4|3|
+|6|2|4|2|4|2|
+|6|1|4|1|4|1|
+
+
+ECN_CGN_TABLE:
+|Outer_DSCP|Outer_ECN|Inner_DSCP|Inner_ECN|Decap_DSCP|Decap_ECN|
+|-|-|-|-|-|-|
+|2|2|3|2|3|3|
+|6|2|4|2|4|3|
+
+
+- PIPE MODE Packet:
+#### IPV4 IN IPV4 Packet <!-- omit in toc --> 
+- expected egress packet = Ether(dst=00:01:01:99:01:01,src=ROUTER_MAC)/IP(dst=192.168.1.1,src=192.168.60.1, ip_dscp=``DSCP COL``, ip_ecn=``INNER_ECN``)/TCP()
+- Ingress encap packet=Ether(dst=ROUTER_MAC)/IP(dst=`STAND_BY_TOR_IP``,src=``ACTIVE_TOR_IP``,ip_dscp=``EXP_DSCP COL``, ip_ecn=``OUTER_ECN``)/IP(src=192.168.60.1,dst=192.168.1.1, ip_dscp=``DSCP COL``, ip_ecn=``INNER_ECN``)/TCP()
+
+
+#### IPV6 IN IPV4 Packet <!-- omit in toc --> 
+- Expected egress packet = Ether(dst=00:01:01:99:01:01,src=ROUTER_MAC)/IP(src=fc02::60:1,dst=fc02::1:1,ip_dscp=``EXP_DSCP COL``, ip_ecn=``INNER_ECN``)/TCP()
+- Ingress encap packet=Ether(dst=ROUTER_MAC)/IP(src=``ACTIVE_TOR_IP``,dst=`STAND_BY_TOR_IP``,ip_dscp=``EXP_DSCP COL``, ip_ecn=``OUTER_ECN``)/IP(src=fc02::60:1,dst=fc02::1:1,ip_dscp=``EXP_DSCP COL``, ip_ecn=``INNER_ECN``)/TCP()
+
+
+### Test steps: <!-- omit in toc --> 
+- decap_dscp_remap:
```

Comment:
There is no DSCP remap at the decap side. Since we are using PIPE mode for DSCP, when the ipinip packets are decapsulated, the inner DSCP value is preserved.

Reply:
change the test cases name to decap_inner_dscp_preserve